__tests__/integration.test.js

```diff
@@ -3,27 +3,34 @@ const seed = require("../db/seeds/seed");
 const data = require("../db/data/test-data/index");
 const request = require("supertest");
 const app = require("../app");
-const Test = require("supertest/lib/test");
 
 beforeAll(() => seed(data));
-afterAll(() => db.end());
+afterAll(() => {
+   return db.end()});
 
 describe("GET /api/topics", () => {
-  test("responds with a 200 status code", () => {
-    return request(app).get("/api/topics");
-    Test.expect(200);
-  });
-  test("Call to endpoint should return array of all topic objects", () => {
-    return db.query(`SELECT * FROM topics;`).then(({ rows: topics }) => {
-      expect(topics).toHaveLength(3);
-      topics.forEach((topic) => {
-        expect(topic).toEqual(
-          expect.objectContaining({
-            description: expect.any(String),
-            slug: expect.any(String),
-          })
-        );
+  test("responds with a 200 status code", function () {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        const output = body.topics;
```

Comment:
I don't think that output is a good variable name here, topics is what it is so you can either call it that through assignment:

`const topics = body.topics`

or destructuring:

`const {topics} = body`